test/blueprinter_test.rb

```diff
@@ -91,13 +91,14 @@ def test_views
     )
   end
 
-  def test_field_with_block
-    my_obj = OpenStruct.new(id: 1, first_name: 'Meg', last_name: 'Ryan')
+  def test_render_with_options
```

Comment:
This test has become confusing to me. We pass in a user and vehicle options to get back a serialized vehicle? What happened to `Meg Ryan` and why do we care about her vehicle?

Additionally, don't we want a test that the block works and a test that options work?

Reply:
Thanks for catching this. I did not mean to over-write the block test with the options test. I meant to leave the `test_field_with_block` alone, and make a separate `test_render_with_options`, but i somehow ended up over-writing the former.